pkg/apis/batch/types.go

```diff
@@ -183,9 +183,11 @@ type JobSpec struct {
 	// for each index.
 	// When value is `Indexed`, .spec.completions must be specified and
 	// `.spec.parallelism` must be less than or equal to 10^5.
+	// In addition, The Pod name takes the form
+	// `$(job-name)-$(index)-$(random-string)`,
```

Comment:
did we actually make this change?

Reply:
https://github.com/kubernetes/kubernetes/pull/101601

Reply:
please mention in the notes to reviewers that the change is being made in a separate PR

Reply:
done